Sources/PrivoSDK/internal/age-gate/PrivoAgeGateInternal.swift

```diff
@@ -88,39 +103,34 @@ internal class PrivoAgeGateInternal {
     
     func getStatusEvent(_ userIdentifier: String?, nickname: String?) async -> AgeGateEvent {
         let agId = storage.getStoredAgeGateId(userIdentifier: userIdentifier, nickname: nickname)
-        let fpId = await storage.getFpId()
         if (agId == nil && nickname != nil) {
             return await processStatus(
                 userIdentifier: nil,
                 nickname: nickname,
-                agId: nil,
-                fpId: fpId
+                agId: nil
             )
         } else {
             return await processStatus(
                 userIdentifier: userIdentifier,
                 nickname: nickname,
-                agId: agId,
-                fpId: fpId
+                agId: agId
             )
         }
     }
     
     func getAgeGateState(userIdentifier: String?, niсkname: String?) async -> AgeState? {
         let agId = storage.getStoredAgeGateId(userIdentifier: userIdentifier, nickname: niсkname)
-        let fpId = await storage.getFpId()
-        do {
-            let settings = try await storage.serviceSettings.getSettings()
-            guard let settings = settings else { return nil }
-            let state = AgeState(fpId: fpId, agId: agId, settings: settings)
-            return state
-        } catch _ {
+        guard let settings = try? await storage.serviceSettings.getSettings() else {
             return nil
         }
+        let state = AgeState(agId: agId, settings: settings)
```

Comment:
From where you are getting settings now?

Reply:
This fragment of code doesn't change any logic with settings: ```storage.serviceSettings.getSettings()``` used for getting value as it was.